package.json

```diff
@@ -34,9 +34,9 @@
     "marked": "^4.0.12",
     "marked-terminal": "^5.1.1",
     "minimist": "^1.2.6",
-    "moment": "^2.29.1",
+    "moment": "^2.29.4",
```

Comment:
@dholmen Når vi nå like gjerne er inne og oppdaterer, går det greit om vi også fjerner alle hatter i samme slengen?

Reply:
Also: Kunne vi prøvd å bytte ut moment med for eksempel date-fns? Moment vil jo ikke bli oppdatert noe særlig lenger.